api/src/main/java/org/apache/cloudstack/api/response/VolumeResponse.java

```diff
@@ -261,6 +261,10 @@ public class VolumeResponse extends BaseResponseWithTagInformation implements Co
     @Param(description = "true if storage snapshot is supported for the volume, false otherwise", since = "4.16")
     private boolean supportsStorageSnapshot;
 
+    @SerializedName(ApiConstants.DELETE_PROTECTION)
+    @Param(description = "true if volume has delete protection.")
```

Comment:
```suggestion
    @Param(description = "true if volume has delete protection.", since = "4.20.0")
```

if targeted for 4.20